src/main/webapp/WEB-INF/static/js/trialmanager/germplasm/add-new-entries-modal.js

```diff
@@ -0,0 +1,91 @@
+(function () {
+	'use strict';
+
+	const module = angular.module('manageTrialApp');
+
+	module.controller('AddNewEntriesController', ['$scope', '$rootScope', '$uibModal', '$uibModalInstance', 'studyEntryService',
+		function ($scope, $rootScope, $uibModal, $uibModalInstance, studyEntryService) {
+
+			const CHECK_ENTRY_ID = 10180;
+
+			$scope.selected = {};
+			$scope.entryTypes = [];
+			$scope.selectedGids = [];
+
+			$scope.init = function () {
+				studyEntryService.getEntryTypes().then(function (entryTypes) {
+					buildEntryTypes(entryTypes);
+				})
+			};
+
+			$scope.cancel = function () {
+				$uibModalInstance.dismiss();
+			};
+
+			$scope.openSelectGermplasmForAddingEntries = function() {
```

Comment:
Have you tried creating a common function to open germplasm selector modal? Similar to `$rootScope.openConfirmModal`

Reply:
Logged this as techinical backlog: https://ibplatform.atlassian.net/browse/IBP-4168